packages/loader/container-utils/src/error.ts

```diff
@@ -3,27 +3,26 @@
  * Licensed under the MIT License.
  */
 
-import { assert } from "@fluidframework/common-utils";
 import {
     ContainerErrorType,
     IGenericError,
     ICriticalContainerError,
     IErrorBase,
+    IThrottlingWarning,
 } from "@fluidframework/container-definitions";
-import { LoggingError } from "@fluidframework/telemetry-utils";
+import { isILoggingError, LoggingError } from "@fluidframework/telemetry-utils";
 import { ITelemetryProperties } from "@fluidframework/common-definitions";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
 
 function messageFromError(error: any): string {
     if (typeof error?.message === "string") {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return error.message;
+        return error.message as string;
     }
-    return `${error}`;
+    return String(error);
```

Comment:
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

Reply:
maybe this usage is ok, but i prefer the old way and not to introduce any upper case String usages

Reply:
The old way throws an exception if `error` is a `Symbol`. This way was recommended on SO or the TS docs or something.